.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -3395,14 +3395,14 @@ stages:
           GITHUB_TOKEN: $(GITHUB_TOKEN)
 
       - bash: |
-          echo "Creating dispatch event for https://github.com/DataDog/dd-trace-dotnet/actions/workflows/docker-base-images.yml with ref=$(Build.SourceBranch) and azdo_build_id=$(Build.BuildId) and is_snapshot=$(isMainBranch)"
```

Comment:
As this jobs run on release branches, does it mean we can override the latest_snapshot with a release build? As we build them daily, we would overwrite them, right?
Hmmm, actually I must be missing something because if what I say is true we were putting "latest" for version from realease branches...
Eureka :p, it works as we don't trigger sheduled jobs for release/1.x anymore. So it works as long as we don't do a new release branch. 

Ok to be future proof, should we move this to its own stage and have a trigger on `isMainBranch` only? Or change the trigger for this stage as it may make sense 🤷 ?

Reply:
I'm confused... 😅 

Basically, I think we can completely ignore 1.x now. There's basically 0% chance we can ever release from it. There's too many extra moving parts and changes that simply aren't in the pipeline in that branch any more.

> Should we move this to its own stage and have a trigger on isMainBranch only

We could 🤷‍♂️ But presumably we want to trigger on hotfixes too no? In which case this change is correct AFAICT?

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/4448/commits/443e1d43d0df9b195a36d164ba14f0c80976bea3